web/components/new-contract/contract-params-form.tsx

```diff
@@ -154,6 +155,19 @@ export function ContractParamsForm(props: {
     ? parseFloat(initialValueString)
     : undefined
 
+  const closeDateMap: { [key: string]: number | string } = {
+    'A day': 1,
+    'A week': 7,
+    '30 days': 30,
+    'This year': daysLeftInTheYear,
+  }
+
+  const NEVER_IN_DAYS = 30 * 12 * 1000 // ~1000 years
+  if (outcomeType == 'POLL') {
+    closeDateMap['Never'] = NEVER_IN_DAYS
+  }
+  const [neverCloses, setNeverCloses] = useState(false)
```

Comment:
Does this need to be a separate useState? Isn't it just:

`const neverCloses = closeDateInDays === NEVER_IN_DAYS`

Reply:
closeDateInDays doesn't exist, it just adds that amount to now